manifests/backports.pp

```diff
@@ -73,10 +80,12 @@
   }
 
   apt::source { 'backports':
-    location   => $location,
-    release    => "${release_real}-backports",
-    repos      => $repos,
-    key        => $key,
-    key_server => 'pgp.mit.edu',
+    location    => $location,
+    release     => "${release_real}-backports",
+    repos       => $repos,
+    key         => $key,
+    key_server  => 'pgp.mit.edu',
+    pin         => $pin_priority,
```

Comment:
This line is causing the test failures. The pin is defined above, so that causes the duplicate resource error. Removing this line (and removing the corresponding line in `spec/classes/backports_spec.rb`) should resolve this issue.